requirements/dev.in

```diff
@@ -1,3 +1,5 @@
 pip-tools
 -r prod.txt
+factory-boy
+ipython
```

Comment:
The cookiecutter adds `factory-boy` and `ipython` to `dev.in` because these are useful to have when debugging things (using Python's `breakpoint()` and standard debugger) or when running `make shell`: you can get IPython instead of the basic Python prompt, and if the app has any test factories you can use them to create objects